projects/dashboards/src/persistence/store/dashboard-remote-store.ts

```diff
@@ -0,0 +1,59 @@
+/* tslint:disable:no-console */
+import { Injectable } from '@angular/core';
+import { EMPTY, Observable, throwError } from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+/* tslint:disable:import-blacklist */
+import { CustomDashboardService } from '../../../../observability/src/pages/custom-dashboards/custom-dashboard.service';
+
+import {
+  DashboardCreationData,
+  DashboardStore,
+  DashboardUpdateData,
+  DashboardUpsertData,
+  PersistedDashboard
+} from './dashboard-store';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DashboardRemoteStore implements DashboardStore {
+  public constructor(private readonly customDashboardService: CustomDashboardService) {}
+  public read(id: string): Observable<PersistedDashboard> {
+    return this.customDashboardService.fetchDashboardConfigById(`/${id}.json`).pipe(
+      catchError(() => throwError(`Provided ID does not exist: ${id}`)),
+      map(
+        /* tslint:disable:object-literal-shorthand */
+        (data): PersistedDashboard => ({
+          tags: [],
+          content: data.json,
+          name: data.location,
+          version: 1,
+          id
+        })
+      )
+    );
+  }
+  public readAll(): Observable<PersistedDashboard> {
+    return EMPTY;
+  }
+  public create(dashboard: DashboardCreationData): Observable<PersistedDashboard> {
+    console.log(dashboard);
+
+    return EMPTY;
+  }
+  public update(dashboard: DashboardUpdateData): Observable<PersistedDashboard> {
+    console.log(dashboard);
+
+    return EMPTY;
+  }
+  public delete(id: string): Observable<void> {
+    console.log(id);
+
+    return EMPTY;
```

Comment:
Can we remove all the `console.log` here?

Reply:
Gives ESLint errors, again not needed in phase 1. All these issues would be solved after phase 1 PR is merged

Reply:
Removing `console.log` is giving lint error?

Reply:
Wouldn't merging this litter the production app's console? Since this is going into the main branch, please remove any debugging code that might affect the production app. Every commit to main should be deployable ideally.